cmd/skywire-cli/commands/config/update/update.go

```diff
@@ -200,6 +201,9 @@ var vpnServerUpdateCmd = &cobra.Command{
 		if addVPNServerPasscode != "" {
 			changeAppsConfig(conf, "vpn-server", "--passcode", addVPNServerPasscode)
 		}
+		if setVPNServerNetIfc != "" {
+			changeAppsConfig(conf, "vpn-server", "-netifc", setVPNServerNetIfc)
```

Comment:
It should be `--netifc` and not `-netifc`

Reply:
Done.